app/ents/employee/endpoints.py

```diff
@@ -70,19 +72,18 @@ def login_employee():
             code=HTTPStatus.OK,
             token=security.create_token(employee),
         )
-
+    
     return error_response(error=EmployeeDoesNotExist.msg, code=HTTPStatus.BAD_REQUEST)
 
 
 @bp.after_request
-# @jwt_required
 def refresh_expiring_jwts(success_response):
     try:
         employee_id = str(get_jwt_identity())
         if employee_id:
             exp_timestamp = get_jwt()["exp"]
             now = datetime.now(timezone.utc)
-            target_timestamp = datetime.timestamp(now + timedelta(minutes=30))
+            target_timestamp = datetime.timestamp(now + timedelta(minutes=2))
```

Comment:
Probably access the value from app.core.settings instead of hard-coding

Reply:
alright. are we using 2 minutes expiration time tho?

Reply:
Yeah